build_tools/ci/run_matmul_test.sh

```diff
@@ -768,3 +749,48 @@ run_matmul_test \
     --lhs_rhs_type "i32" \
     --acc_type "i32" \
     --m "32" --k "32" --n "32"
+
+###################################################################
+# Chess tests
+###################################################################
+
+run_matmul_test \
+    --name_prefix "chess_i32_matmul" \
+    --lhs_rhs_type "i32" \
+    --acc_type "i32" \
+    --m "64" \
+    --n "64" \
+    --k "64" \
+    --num_repeat_runs "0" \
```

Comment:
not sure if this is intentional since in the PR description its mentioned e2e works. with ` --num_repeat_runs "0"` we are not actually running only compiling?

Reply:
:facepalm: i thought it meant no repeats but just the first run. ok let me remove (and :crossed_fingers: everything still works).

Reply:
all good